TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/NoteSelectionListener.kt

```diff
@@ -1,15 +1,40 @@
 package com.microsoft.device.display.samples.twonote
 
+import android.content.Context
+import android.util.Log
 import android.view.ActionMode
 import android.view.Menu
 import android.view.MenuItem
 import android.widget.AbsListView
+import android.widget.ArrayAdapter
 import android.widget.ListView
+import androidx.core.view.get
+import androidx.core.view.iterator
+import androidx.core.view.size
+import com.microsoft.device.display.samples.twonote.model.INode
+import java.io.File
```

Comment:
I think there are some unused imports here

Reply:
(and some other ktlint issues haha)

Reply:
one day I'll remember to lint before pushing

Reply:
that's what the pipeline will be for 😉